data/core/nagview.lua

```diff
@@ -91,7 +91,7 @@ function NagView:each_option()
 
     for i = #self.options, 1, -1 do
       opt = self.options[i]
-      bw = opt.font:get_width(opt.text) + 2 * BORDER_WIDTH + style.padding.x
+      bw = style.font:get_width(opt.text) + 2 * BORDER_WIDTH + style.padding.x
```

Comment:
At this point `style.font:get_width(opt.text)` could be moved outside the `for`.

Reply:
Ok, nevermind, was still thinking about `:get_height`...